addons/website_sale/static/src/js/website_sale.js

```diff
@@ -69,7 +69,7 @@ publicWidget.registry.websiteSaleCartLink = publicWidget.Widget.extend({
         clearTimeout(timeout);
         $(this.selector).not(ev.currentTarget).popover('hide');
         timeout = setTimeout(function () {
-            if (!self.$el.is(':hover') || $('.mycart-popover:visible').length) {
```

Comment:
I think that since we are triggering an event and not actually hovering the mouse, this was preventing the popover to be poped

Reply:
The issue with this change is that it will display the popover on mouse enter whether or not we are still on the cart icon

Reply:
You can extract the function to check whether we should still display the popover or not and patch it before the tour, see `google_analytics_view_item`